vcl_templates/www.vcl.erb

```diff
@@ -220,6 +220,10 @@ sub vcl_recv {
     set req.http.GOVUK-ABTest-EducationNavigation = "A";
   } else if (req.url ~ "[\?\&]ABTest-EducationNavigation=B(&|$)") {
     set req.http.GOVUK-ABTest-EducationNavigation = "B";
+  } else if (req.url ~ "^\/education(\/)?.*$") {
```

Comment:
Could the regex be simplified to `^/education/` in both places? It's doing a similar thing to line 310, and we don't bother escaping the forward slashes or matching the full path there.